src/main/resources/native/android/c/glibc_shim.c

```diff
@@ -1,3 +1,30 @@
+/*
+ * Copyright (c) 2019, 2020, Gluon
```

Comment:
These files were introduced in 2020, so no need to add 2019?

Reply:
fixed